### DISCUSSION TITLE
bookmarks import in 0.21.2

### DISCUSSION THREAD

**Discussion by castropic** [Stellarium/stellarium#1961] [Q&A]
Hello,

my bookmarks have disappeared since the update 0.21.2.
I also cannot import the bookmarks.json.
Is there a solution for this?
In which older version does it still work and where can I download it? 

greetings Carsten

**Comment by gzotti** [comment]
Discussed in #1944. Somebody is working on it.

**Comment by Menno5** [comment]
Carsten
With the previous version 0.21.1 the bookmarks work. They are working on 0.21.2 to correct it but for the time being you could revert back to 0.21.1. I use that for now too.
Download is here: [https://github.com/Stellarium/stellarium/releases/tag/v0.21.1](https://github.com/Stellarium/stellarium/releases/tag/v0.21.1)

**Reply by castropic** [reply]
thank you I will do that

**Comment by axd1967** [comment]
just remember that there is no guarantee that files of an updated version of Stellarium can be read by an older version.

consider backing up your config directory before installing an older version, just in case.

**Reply by castropic** [reply]
yes, I had already done this before the last update. I also needed this ;)
